packages/manager/modules/iam/src/dashboard/users/sso/details/sso-details.html

```diff
@@ -1,7 +1,7 @@
 <oui-header heading="{{:: 'user_users_sso_detail_title' | translate }}">
 </oui-header>
 <div>
-    <a class="col-md-12 mb-3" data-ng-href="{{'#/useraccount/users' }}">
+    <a class="col-md-12 mb-3" data-ng-click="$ctrl.goToUsers()">
```

Comment:
The link (`<a>`) should to have href tag.
Or maybe use the ui-kit web component back button https://ovh.github.io/ovh-ui-kit/?path=/docs/design-system-components-back-button-webcomponent--simple